### PR TITLE
chore: update issue labels

### LINKED ISSUE #571: Update issue templates to use status/triage for backstage-plugins
### What needs to be done?
Modify the github issue templates in backstage-plugins to use the `status/triage` label.
Remove the ability to create blank issues in favor of a blank issue template that automatically uses the `status/triage` label.

### PR DESCRIPTION
**Pull request by Zaperex**
### Description
Updated the existing issue templates to also use the `status/triage` label.
Removed the ability for users to create blank issues in favor of creating a blank issue template to enforce the usage of `status/triage` for all issues.

### Which issue(s) does this PR fix
Fixes #571

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: ❓ Janus IDP Community Slack
+    url: https://janus-idp.slack.com/archives/C04EDTPJRK5
```
Comment (tumido) [inline .github/ISSUE_TEMPLATE/config.yml:4]: Can we use an invite link instead? This link is useless unless we let people join the slack workspace.

https://join.slack.com/t/janus-idp/shared_invite/zt-1pxtehxom-fCFtF9rRe3vFqUiFFeAkmg

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
```
Comment (schultzp2020) [inline .github/ISSUE_TEMPLATE/config.yml:1]: Can we enable blank issues?

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (Zaperex) [reply .github/ISSUE_TEMPLATE/config.yml:4]: will do

**Review by tumido — APPROVED** [review]
/lgtm
/approve

**Pull request merged**